test/account_tree.test.ts

```diff
@@ -37,7 +41,7 @@ describe("Account Tree", async () => {
     });
     it("update with single leaf", async function() {
         for (let i = 0; i < 33; i++) {
-            const leaf = web3.utils.randomHex(32);
+            const leaf = ethers.utils.hexlify(ethers.utils.randomBytes(32));
```

Comment:
`ethers.utils.randomBytes` returns Uint8Array for bytes manipulation. But the current ts utils are written to support strings only. And we are doing all the weird `slice` and `"0x" +` to manipulate hex strings.
I'll push more commits to fix this.

Reply:
left the task here https://github.com/thehubbleproject/RedditHubble/issues/186